tools/api-builder/angular.io-package/templates/class.template.html

```diff
@@ -67,7 +67,7 @@
 .div(class="row-margin" layout="row" layout-xs="column")
   div(flex="20" flex-xs="100")
     h2(class="h2-api-docs") Class Description
-  div(class="class-description-content" flex="80" flex-xs="100")
+  div(class="code-links" flex="80" flex-xs="100")
```

Comment:
@naomiblack would it be possible to re-merge this branch in? I force pushed this, probably after you merged in. Otherwise, I can piggy back these changes in my next PR.